integration/src/test/scala/ammonite/integration/BasicTests.scala

```diff
@@ -30,7 +30,7 @@ object BasicTests extends TestSuite{
       )
     'hello{
       val evaled = exec('basic/"Hello.sc")
-      assert(evaled.out.trim == "Hello World")
+      assert(evaled.out.trim.replace("Compiling Script\n", "") == "Hello World")
```

Comment:
For this, and the other tests that are broken, can we pass in `-s` rather than doing a string-replace check?